Web/MetagurukulAccountregistration.html

```diff
@@ -0,0 +1,167 @@
+							<option>August</option>
+							<option>September</option>
+							<option>October</option>
+							<option>November</option>
+							<option>December</option>
+						</select>
+						<input type="text" name="day" placeholder="Day" style="width: 25%;" />
+						<input type="text" name="Year" placeholder="Year"style="width: 25%;" />
+					</td>
+				</tr>
+				<tr>
+					<td colspan="2" align="right">Country</td>
+					<td colspan="2">
+						<select name="Gender" style="width: 91%;">
+							<option>Select</option>
+							<option>Australia</option>
+							<option>Argentina</option>
+							<option>India</option>
+							<option>Srilanka</option>
+							<option>UAE</option>
+							<option>USA</option>
+					</select></td>
+				</tr>
+				<tr>
+					<td colspan="4">
+						<b>
+							<font size="5">Select an ID and Password</font>
+						</b>
+					</td>
+				</tr>
+				<tr>
+					<td colspan="5">
+						<hr>
+					</td>
+				</tr>
+				<tr>
+					<td colspan="2" align="right">Metagurukul ID</td>
+					<td colspan="2">
+						<input type="text" name="day" /> 
+						@ 
+						<select name="email" style="width: 40%">
+							<option>Select One</option>
+							<option>metagurukul.com</option>
+							<option>metacube.com</option>
+						</select>
+						<button type="check" name="check">Check</button>
+					</td>
+				</tr>
+				<tr>
+					<td colspan="2" align="right">Password</td>
+					<td>
+						<input type="text" name="password" style="width: 130%" />
+					</td>
+				</tr>
+				<tr>
+					<td colspan="2" align="right">Re-type Password</td>
+					<td>
+						<input type="text" name="password" style="width: 130%" />
+					</td>
+				</tr>
+				<tr>
+					<td colspan="4">
+						<b>
+							<font size="5">In case you forgot your ID or Password</font>
+						</b>
+					</td>
+				</tr>
+				<tr>
+					<td colspan="5">
+						<hr>
+					</td>
+				</tr>
+				<tr>
+					<td colspan="2" align="right">Alternate email(optional)</td>
+					<td colspan="2">
+						<input type="email" name="alternateemail"style="width: 85%" />
+					</td>
+				</tr>
+				<tr>
+					<td colspan="2" align="right">Select Question 1</td>
+					<td colspan="2">
+						<select name="question1" style="width: 85%">
+							<option>Select One</option>
+							<option>What is your mother's name</option>
+							<option>What is your father's name</option>
+						</select></td>
+					<td>
+				</tr>
+				<tr>
+					<td colspan="2" align="right">Your answer</td>
+					<td>
+						<input type="text" name="answer1" style="width: 175%" />
+					</td>
+				</tr>
+				<tr>
+					<td colspan="2" align="right">Select Question 2</td>
+					<td colspan="2">
+						<select name="question2" style="width: 85%">
+							<option>Select One</option>
+							<option>What is your mother's name</option>
+							<option>What is your father's name</option>
+						</select></td>
+					<td>
+				</tr>
+				<tr>
+					<td colspan="2" align="right">Your answer</td>
+					<td>
```

Comment:
ACTION: Why you not used colspan here to merge 2 columns? if you will use it and will put text box in that column you won't need to give width greater than 100%.